modules/features/settings/src/main/res/layout/fragment_settings_appearance.xml

```diff
@@ -251,6 +251,56 @@
                 app:layout_constraintStart_toStartOf="@+id/lblUseEmbeddedArtwork"
                 app:layout_constraintTop_toBottomOf="@+id/lblUseEmbeddedArtwork" />
 
+            <View
```

Comment:
I believe this setting should be above already existing `Use embedded MP3 artwork` because RSS artwork takes precedence over it.